app/src/main/java/com/github/polypoly/app/ui/game/PropertyInteractUIComponent.kt

```diff
@@ -80,10 +78,15 @@ private fun PropertyInteractButtons(showBuyDialog: MutableState<Boolean>) {
             Text(text = "Bet")
         }
         Button(
-            onClick = { interactingWithProperty.value = false },
+            onClick = { leaveBetDialog() },
             modifier = Modifier.testTag("closeButton")
         ) {
             Text(text = "Close")
         }
     }
 }
+
+private fun leaveBetDialog() {
+    interactingWithProperty.value = false
+    mapViewModel.currentPlayer!!.playerState.value = PlayerState.INTERACTING
```

Comment:
You can use `mapViewModel.currentPlayer?.playerState?.value = PlayerState.INTERACTING` instead, to avoid `!!`